gomint-server/src/main/java/io/gomint/server/world/block/Log.java

```diff
@@ -86,6 +88,15 @@
     private static final AxisBlockState AXIS = new AxisBlockState(() -> new String[]{"pillar_axis"});
     private static final BooleanBlockState STRIPPED = new BooleanBlockState(() -> new String[]{"stripped_bit"});
 
+    @Override
+    public boolean beforePlacement(EntityLiving<?> entity, ItemStack<?> item, Facing face, Location location) {
+        super.beforePlacement(entity, item, face, location);
+        STRIPPED.detectFromPlacement(this, entity, item, face);
+        VARIANT.detectFromPlacement(this, entity, item, face);
```

Comment:
This should be done from the item implementations `block()` to ensure that the item to block transition is correct